### PR TITLE
fixed doxygen docs for one subroutine

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #15 

I needed a doxygen example for a presentation so I fixed a random subroutine. ;-)

### CODE REVIEW

File: src/getntbe.f [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
-C>   THE SPECIFIED ASCII MASTER TABLE B OR MASTER TABLE D FILE.  THIS
-C>   LINE CONTAINS, AMONG OTHER THINGS, THE FXY NUMBER CORRESPONDING TO
-C>   THIS ENTRY.
+C> This subroutine gets the first line of the next entry in
+C> the specified ascii master table b or master table d file. This
+C> line contains, among other things, the fxy number corresponding to
+C> this entry.
 C>
-C> PROGRAM HISTORY LOG:
-C> 2007-01-19  J. ATOR    -- ORIGINAL AUTHOR
```
Comment (edwardhartnett) [inline src/getntbe.f:10]: @jbathegit I am removing these PROGRAM HISTORY LOG sections when they contain nothing but the initial creation of the subprogram. That's already captured in the author and date tags, so no need to repeat it.

Whether we should continue updating these history logs is unclear. This is what github does already. Do we need to continue doing it? That question has been asked and I don't have any strong answer... Do you?

File: src/getntbe.f [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
-C>                NEXT TABLE ENTRY
-C>     LINE     - CHARACTER*(*): FIRST LINE OF NEXT TABLE ENTRY
-C>     IRET     - INTEGER: RETURN CODE:
-C>                       0 = normal return
-C>                      -1 = end-of-file encountered while reading
-C>                           from LUNT
-C>                      -2 = I/O error encountered while reading
-C>                           from LUNT
-C>
-C> REMARKS:
```
Comment (edwardhartnett) [inline src/getntbe.f:28]: I am also removing all text elements like REMARKS:, ABSTRACT, etc. These may have been helpful in Fortran comment formatting, but are not needed for Doxygen documentation, which controls formatting without needing these.

File: src/getntbe.f [hunk 3]
```diff
@@ -1,36 +1,24 @@
...
-C>    THIS ROUTINE CALLS:        BORT2    IGETNTBL IGETFXY  IFXY
-C>                               PARSTR
-C>    THIS ROUTINE IS CALLED BY: RDMTBB   RDMTBD   RDMTBF
-C>                               Normally not called by any application
-C>                               programs.
+C> This routine calls: bort2() igetntbl() igetfxy() ifxy parstr()
+C> This routine is called by: rdmtbb() rdmtbd() rdmtbf()
+C> Normally not called by any application programs.
```
Comment (edwardhartnett) [inline src/getntbe.f:21]: Also note that whitespace is not significant for doxygen, so spacing things out to a certain column is not needed. We can use lists and tables to achieve a more organized effect when desire it.

Comment (jbathegit) [reply src/getntbe.f:21]: Understood

**Review by jbathegit — APPROVED** [review]

**Comment by edwardhartnett** [comment]
@jbathegit the date field should be the date the subroutine or function was first added, not the date it was most recently updated.

I will keep the PROGRAM HISTORY LOG from now on, even if it has only one entry. Should we lower-case it?

As noted in the call. the breif tag can be automatic everywhere. I will make sure that is turned on.

**Comment by jbathegit** [comment]
IMHO, it's more useful and informative if the date is the date of the most recent update to the subprogram, not the date that the first version was written.  But you seem to be implying that the reverse is the industry standard, and if that's the case then I'll accept that approach and make the changes where needed - just let me know.

Yes, I'm fine with lowercasing the "PROGRAM HISTORY LOG" label going forward, and also the "THIS ROUTINE CALLS/IS CALLED BY" as well, though I may add <b> (i.e. bold) tags to them to allow them to still stand out a bit.

re: the brief tag, how does Doxygen know which part of the documentation to use for the brief?  Does it always just default to grabbing the first sentence or two following the `@file` section?

**Comment by edwardhartnett** [comment]
Yes, the brief tag will be the first sentence of the documentation block, for files, subroutines, vars, etc. This works well and encourages good documentation practices.

Yes, the date field always refers to the date the function was added, that's how it's used everywhere. The most recent modified date is found in the github repo.

**Comment by jbathegit** [comment]
OK, got it, though I will point out that when users are reading through the github-io documentation, they may not always also be looking back at the github repo ;-)

re: the brief tag, please let me know when you have the automatic capability turned on, and at that point I'll stop using it in all of the routines.

**Pull request merged**